velox/type/Tokenizer.h

```diff
@@ -51,6 +54,9 @@ class Tokenizer {
   const char UNICODE_CARET = '^';
 
   const std::string path_;
+  // Field name separators by default.
+  std::vector<char> separators_ =
```

Comment:
Can we use a struct to make it more readable?

```c++
struct Separators {
  char dot;
  char quote;
  // ...
};
```

Reply:
Thanks for your review, but I am confusing here.
Is it suggested to use struct to replace `std::vector`? But how to check if a char is contained in it, by verifying, e.g., if `dot == ""`? Also, to customize new separator in it seems impossible.

Reply:
You can set dot = '\0' for example to disable it.  This is just interface though, in the implementation you can check them one by one (won't be slower than checking `vector`), or keep a bit mask of 256 bits to check whether some character is contained in it.  To customize these characters, you will also need to replace all `DOT`, `OPEN_BRACKET` with values in this struct.

Reply:
Thanks for your advide. Understood, and I will modify as you suggest.

Reply:
@Yuhta Changed to struct as suggested. Could you review again? Thanks.